product_multi_image/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# © 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Multiple Images in Products",
+    "version": "16.0.1.0.0",
+    "author": "Antiun Ingeniería, "
+    "Tecnativa, "
+    "LasLabs, "
+    "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Sales Management",
+    "pre_init_hook": "post_init_hook",
```

Comment:
The pre ulinit hook, is triggering a post init hook? A bit confusing.. Is that correct?

Reply:
Thanks for catching this one, this slipped from previous version and culprit of some headaches